controllers/middlewares/access_token.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const log = require('winston');
+
+const auth = require('helpers/auth_helper');
+const { AuthorizationError } = require('errors');
+
+const jwt = require('lib/jwt');
+const responses = require('lib/responses');
+
+module.exports = async (req, res, next) => {
```

Comment:
why do we have `next` here?

Reply:
@wildtron Since there are no other middlewares being called, we can remove the `next`, but will that be fine? Since `async` with `next` is just the same implementation in the `Admin Tool BE`.

Reply:
That's not it, we can remove the next because we're letting router_extended do the handling of `next`